crates/rspack_core/src/dependency/mod.rs

```diff
@@ -208,12 +209,19 @@ impl ExportSpec {
   }
 }
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
```

Comment:
![image](https://github.com/web-infra-dev/rspack/assets/17974631/3ea78358-6bbd-4ad0-9643-b0079a1ae8a9)
```rs
#[derive(Debug, Default)]
pub struct ExportSpec {
  pub name: JsWord,
  pub export: Option<Vec<JsWord>>,
  pub exports: Option<Vec<ExportNameOrSpec>>,
  pub can_mangle: Option<bool>,
  pub terminal_binding: Option<bool>,
  pub priority: Option<u8>,
  pub hidden: Option<bool>,
  pub from: Option<ModuleGraphConnection>,
  pub from_export: Option<ModuleGraphConnection>,
}

impl ExportSpec {
  pub fn new(name: String) -> Self {
    Self {
      name: JsWord::from(name),
      ..Default::default()
    }
  }
}

#[derive(Debug)]
pub enum ExportNameOrSpec {
  String(JsWord),
  ExportSpec(ExportSpec),
}

impl Default for ExportNameOrSpec {
  fn default() -> Self {
    Self::String(JsWord::default())
  }
}

#[derive(Debug, Default)]
pub enum ExportsOfExportsSpec {
  True,
  #[default]
  Null,
  Array(Vec<ExportNameOrSpec>),
}

```
Here is my version.